plugins/modules/template_workflow_manager.py

```diff
@@ -1850,6 +1739,17 @@ def get_containing_templates(self, containing_templates):
 
             containingTemplates[i].update({"name": name})
 
+            template_details = self.get_templates_details(name)
+            template_details = template_details.get("response")
```

Comment:
If we are not doing any check, can't we directly write as:
template_details = self.get_templates_details(name).get("response")